utils/deploy_netchecker.sh

```diff
@@ -10,7 +10,7 @@ fi
 kubectl get nodes
 
 echo "Installing netchecker server"
-echo '
+cat << EOF > {{ kube_config_dir }}/netchecker-server-pod.yml
```

Comment:
Maybe store location for k8s ymls in some variable w/ default value but which also can be populated by an environment variable if present?